jujugui/static/gui/src/app/components/store/store.js

```diff
@@ -23,33 +23,820 @@ YUI.add('store', function() {
+                </li>
+                <li className="three-col featured-entity">
+                    <a className="featured-entity__link"
+                      href="/?store=jenkins">
+                        <div
+                          className="one-col no-margin-bottom">
+                            <img
+                              src={this._generateIconPath('jenkins')}
+                              alt=""
+                              className="featured-entity__image" />
+                        </div>
+                        <div
+                          className="two-col last-col no-margin-bottom">
+                            <h3 className="featured-entity__title">
+                              Jenkins
+                            </h3>
+                            <p>by Creative</p>
+                        </div>
+                    </a>
+                </li>
+            </ul>
+        </div>
+      </div>);
+    },
+
+    /**
+      The content for the featured charms and bundles section of the page
+
+      @method _featuredSection
+      @return {Object} The contents of the section
+    */
+    _featuredSection: function() {
+      return (<div className="row equal-height">
+          <div className="box box--kibana align-center four-col">
+              <img src={this._generateLocalImagePath('kabana-promo.png')}
+                alt="" className="box__image" />
+              <div className="align-bottom">
+                  <h2>Kibana</h2>
+                  <a href="/?store=kibana"
+                    className="button--inline-neutral">
+                    View the charm
+                  </a>
+              </div>
+          </div>
+          <div className="box box--kubernetes align-center four-col">
+              <img src={this._generateLocalImagePath('kubernetes-promo.png')}
+                alt="" className="box__image" />
+              <div className="align-bottom">
+                  <h2>Observable Kubernetes</h2>
+                  <a href="/?store=observable-kubernetes"
+                    className="button--inline-neutral">
+                    View the bundle
+                  </a>
+              </div>
+          </div>
+          <div
+            className="box box--openstack align-center four-col last-col">
+              <img src={this._generateLocalImagePath('openstack-promo.png')}
+                alt="" className="box__image" />
+              <div className="align-bottom">
+                  <h2>OpenStack</h2>
+                  <a href="/?search=openstack&type=bundle"
+                    className="button--inline-neutral">
+                    View bundles
+                  </a>
+              </div>
+          </div>
+      </div>);
+    },
+
+    /**
+      Generate contents of the tagged list
+
+      @method _tagsSection
+      @return {Object} A generated section listing topics with counts
+    */
+    _tagsSection: function() {
+      var topics = [
```

Comment:
No there isn't a issue but mention of getting a count endpoint from the API. I personally think they should be removed until there generated as they are going out of date already.